pallets/ddc-verification/src/lib.rs

```diff
@@ -1431,9 +1431,9 @@ pub mod pallet {
 						payers: customers_activity_batched[i]
 							.iter()
 							.map(|activity| {
+								let customer_id = activity.clone().customer_id.into_bytes();
 								let account_id =
-									T::AccountId::decode(&mut &activity.customer_id.as_bytes()[..])
-										.unwrap();
+									T::AccountId::decode(&mut &customer_id[..]).unwrap();
```

Comment:
I do not suggest using `unwrap()` in runtime calls, it is better to respond with a `DispatchError` instead of panicking in the runtime

Reply:
@yahortsaryk I will try to fix in my next PR.